### PR TITLE
Added an alternate image for darkmode (#1ym0jx2)

### PR DESCRIPTION
**Pull request by azkyakhan**
https://user-images.githubusercontent.com/82817616/149466887-8792243c-1767-4f8d-a297-9e5dc0b3bcf7.mp4

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -135,7 +136,7 @@ export default defineComponent({
 
 <style scoped>
 
-figure {
+figure{
```
Comment (dt2patel) [inline src/views/Home.vue]: usually we keep a space between the bracket and the selector. Why did you remove it here?

**Review by dt2patel — CHANGES_REQUESTED** [review]
One change

File: src/views/Home.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Home.vue]: Yes, I was using another tag here so due to that may be this happened. I am updating it sir

Comment (azkyakhan) [reply src/views/Home.vue]: Updated sir
